### PR TITLE
feature file for the add contact created 

### PR DESCRIPTION
**Pull request by ritaadhikari**
### Description 
Here the different scenarios of the add contact feature has been added to the addcontact.feature file.

### Issues addressed:
https://github.com/SagarGi/FeatureFiles/issues/4

### CODE REVIEW

File: Features2/tests/features/addcontact.feature [hunk 1]
```diff
@@ -0,0 +1,34 @@
+Feature:AddContact 
+    As a user
+    I want to add my contacts to the ContactFox
+    So can easily remember the contacts that are necessary for message
+
+    Background: User is looged In
+        Given the  user  navigated to the login page
+        When the user submit username and password
+        Then the user should be logged in to the homepage
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]: background does not contains `When` and `Then` step but only `Given` step

File: Features2/tests/features/addcontact.feature [hunk 2]
```diff
@@ -0,0 +1,34 @@
+Feature:AddContact 
+    As a user
+    I want to add my contacts to the ContactFox
+    So can easily remember the contacts that are necessary for message
+
+    Background: User is looged In
+        Given the  user  navigated to the login page
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]:
```suggestion
        Given the user has navigated to the login page
```

File: Features2/tests/features/addcontact.feature [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+        When the user submit username and password
+        Then the user should be logged in to the homepage
+
+        Scenario:add to contacts with personal contact type
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]:
```suggestion
        Scenario: Add to contacts with personal contact type
```

File: Features2/tests/features/addcontact.feature [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        Given the user is navigated to the home page
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]:
```suggestion
        Given the user has navigated to the home page
```

File: Features2/tests/features/addcontact.feature [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+        When the user submit name with "name" and email with "valid email" and chooses the contact type as "personal"
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]: use `datatable`

File: Features2/tests/features/addcontact.feature [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+        Then the user info will be added to the contact list  with the personal  badge
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]: which user to check ? may be give user information that has been added to contact

File: Features2/tests/features/login.feature [hunk 7]
```diff
@@ -6,7 +6,7 @@ Feature:login feature
 
     Scenario: Login to the ContactFox with Valid credentials
         Given a user  has navigated to the login dashboard
-        When  the user enters the valid credentials with email "valid email" and Password "valid password"
+        When  the user enters the valid credentials with email "valid email" and Password "valid password"S
```
Comment (SagarGi) [inline Features2/tests/features/login.feature]:
```suggestion
        When  the user enters the valid credentials with email "valid email" and Password "valid password"
```

File: Features2/tests/features/addcontact.feature [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+ 
+        Scenario:add to contacts with professional contact type
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]:
```suggestion
        Scenario: Add to contacts with professional contact type
```

File: Features2/tests/features/addcontact.feature [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+        Given the user is navigated to the home page
+        When the user submit name with "name" and email with "valid email" and chooses the contact type as "professional"
+        Then the user info will be added to the contact list  with the professional badge
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]: follow same as i have commented

File: Features2/tests/features/addcontact.feature [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+        Scenario Outline:when you input invalid email
+        Given a user has navigated to the homepage
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]:
```suggestion
        Given the user has navigated to the homepage
```

File: Features2/tests/features/addcontact.feature [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+        When the user enters with a "<email>"
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]:
```suggestion
        When the user enters invalid email "<email>"
```

File: Features2/tests/features/addcontact.feature [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+        Then the user will thrown a "<message>"
```
Comment (SagarGi) [inline Features2/tests/features/addcontact.feature]: huh? user will thrown? please make a valid step

File: Features2/tests/features/addcontact.feature [hunk 13]
```diff
@@ -0,0 +1,27 @@
...
+
+        Scenario Outline:when you input invalid email
+        When the user enters with a "<email>"
+        Then the user will be given  a "<message>"
+        Examples:
+        | email            | message|
+        | abchefg98gmailcom |  Please enter an email address|
+        | rits98.com       | Please enter an email address|
+        | rira@gmail       | Please enter an email address|
+        | risahgdgajda36gmail.com | Please enter an email address|
```
Comment (kiranadh1452) [inline Features2/tests/features/addcontact.feature]: Use formatting.

File: Features2/tests/features/addcontact.feature [hunk 14]
```diff
@@ -0,0 +1,27 @@
+Feature:AddContact 
+    As a user
+    I want to add my contacts to the ContactFox
+    So can easily remember the contacts that are necessary for message
+
+    Background: User is looged In
+        Given the  user has  navigated to the login page
+        And the user submits username and password
+        And  the user was logged in to the homepage
```
Comment (kiranadh1452) [inline Features2/tests/features/addcontact.feature]: Make this a single step.

**Review by kiranadh1452 — CHANGES_REQUESTED** [review]

**Review by kiranadh1452 — APPROVED** [review]

**Pull request merged**